### PR TITLE
Fix reals for inactive_hp_thread_queue

### PR DESCRIPTION
**Pull request by dcristoloveanu**

### CODE REVIEW

**Review by BertKleewein — DISMISSED** [review]
:shipit:

File: tests/reals/real_inactive_hp_thread_queue.c [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #include "c_pal/gballoc_hl_redirect.h"
```
Comment (anporumb) [inline tests/reals/real_inactive_hp_thread_queue.c:7]: why is this needed here?

**Review by anporumb — DISMISSED** [review]
:shipit:

File: tests/reals/real_inactive_hp_thread_queue.c (hunk 1 above)
Comment (dcristoloveanu) [reply tests/reals/real_inactive_hp_thread_queue.c:7]: Likely not needed, I am removing it and let's see where it craps out.

**Review by anporumb — APPROVED** [review]
:shipit:
